tests/core/test_resolving.py

```diff
@@ -13,3 +13,22 @@
 # along with Blue Brain Nexus Forge. If not, see <https://choosealicense.com/licenses/lgpl-3.0/>.
 
 # Placeholder for the test suite for selection of target or resolver from input and configuration.
+
+from kgforge.core import KnowledgeGraphForge
+
+
+class TestResolverAsDict:
+    """
+    Tests the .resolver(as_dict=True) function that returns a dictionary
+    """
+
+    def test_resolver_returns_dict(self, config):
+        forge = KnowledgeGraphForge(config)
+        resolvers_dict = forge.resolvers(as_dict=True)
+        assert type(resolvers_dict) is dict
+
+    def test_resolver_returns_correct_object(self, config):
+        dict_result = {'terms': {'sex': {'source': 'sex.json'}}}
```

Comment:
Looking again at the forge config, I think we should use 'bucket' instead of 'source' to avoid clashes with the actual source defined in the forge config.

Reply:
I changed the key and pushed the changes